client/store/index.js

```diff
@@ -0,0 +1,42 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import axios from 'axios';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+
+const LOAD_USERS = 'LOAD_USERS';
+
+//Create Action Creators & Thunks
+
+//Load Users Action Creator
```

Comment:
Would you be able to put your User action creator, thunk, and reducer in a seperate file? That way the index does get super long

Reply:
Sure, and sorry - I just saw that something is off because npm start isn't producing the home page. Let me just check into it for a sec